model/Core/Vocabularies/RelationshipType.md

```diff
@@ -66,6 +66,16 @@ Build Profile specific RelationshipType descriptions can be found [here](https:/
 - buildInvokedBy: Agent that invoked the build
 - buildOnBehalfOf: Actor for which buildInvokedBy is acting on behalf of
 - buildHostOf: Element in which the build instance runs on
+- advisory: (Security) Used to associate a security vulnerability with an element
+- coordinatedBy: (Security) TODOdescription
+- exploitCreatedBy: (Security) TODOdescription
+- fixedBy: (Security) Designates a vulnerability has been fixed by an element, actor or entity
+- foundBy: (Security) Designates an actor or entity was the original discoverer of a security exploit
+- hasResponsibleParty: (Security) TODOdescription
+- issuedBy: (Security) Designates the actor or entity to issue an identity (i.e. CVE ID) for a vulnerability
+- publishedBy: (Security) Designates the actor or entity to make a CVE record available for public use or reference
```

Comment:
Should all of these be in the past tense as we are capturing information already researched and disclosed?
```suggestion
- publishedBy: (Security) Designates the actor or entity that made a vulnerability record available for public use or reference
```